ansible_ai_connect/ai/api/views.py

```diff
@@ -1021,7 +1023,7 @@ def post(self, request) -> Response:
         )
 
         answer = {
-            "role": anonymized_role,
+            "name": anonymized_role,
```

Comment:
It seems a little _weird_ that `name` is the anonymised `role`.. what's the reasoning?

Reply:
`name` is the name of the role, `role` was the former name of the key. I renamed it `name` for consistency since it's also the name of the WCA key.
The key is used as parameter and in the answer.